arviz/plots/ppcplot.py

```diff
@@ -157,11 +160,20 @@ def plot_ppc(
 
         >>> az.plot_ppc(data, num_pp_samples=30, random_seed=7)
     """
-    for group in ("posterior_predictive", "observed_data"):
-        if not hasattr(data, group):
-            raise TypeError(
-                '`data` argument must have the group "{group}" for ppcplot'.format(group=group)
-            )
+    if group == "posterior":
```

Comment:
We should start by checking `group in ("prior", "posterior"):` and raise an error if it is not. 

This would also allow to simplify the attribute checking by using `for groups in ("{}_predictive".format(group), "observed_data"):...`

Reply:
Yeah and it looks much cleaner. Thanks!